main/SAPI.c

```diff
@@ -554,7 +554,24 @@ static int sapi_extract_response_code(const char *header_line)
 
 	for (ptr = header_line; *ptr; ptr++) {
 		if (*ptr == ' ' && *(ptr + 1) != ' ') {
-			code = atoi(ptr + 1);
+			/* beginning of the response code */
+			char *status_description = estrdup(ptr + 1);
+			char *response_code = strtok(status_description, " ");
+
+			code = atoi(response_code);
```

Comment:
Have you considered to use [`ZEND_STRTOUL()`](https://heap.space/search?project=php-src&full=&defs=ZEND_STRTOUL&refs=&path=&hist=&type=&si=defs) instead?

Reply:
Oh, I haven't thought about it. Then code will be much simple. Let me fix!